src/dosbox.cpp

```diff
@@ -666,15 +666,17 @@ void DOSBOX_Init()
 	pstring->Set_help("CPU core used in emulation ('auto' by default). 'auto' will switch to dynamic\n"
 	                  "if available and appropriate.");
 
-	const char* cputype_values[] = { "auto", "386", "386_slow", "486_slow", "pentium_slow",
-#if C_MMX
-	                                  "pentium_mmx",
-#endif
-	                                  "386_prefetch", nullptr};
-	pstring = secprop->Add_string("cputype", always, "auto");
+	const char* cputype_values[] = { "386", "486", "pentium", "pentium_mmx",
+	                                 "386_prefetch", "auto", nullptr};
+	pstring = secprop->Add_string("cputype", always, "pentium_mmx");
 	pstring->Set_values(cputype_values);
 	pstring->Set_help(
-	        "CPU type used in emulation ('auto' by default). 'auto' is the fastest choice.");
+	        "CPU type used in emulation ('pentium_mmx' by default).\n"
```

Comment:
Is it wise to switch from `auto` to `pentium_mmx` if `auto` is the fastest? I'm thinking of backward compatibility.

I don't see any mentions of benchmarking in the comments. Are you sure `pentium_mmx` is just as fast as `auto` for non-MMX programs? I guess we'd need to benchmark first before we change the default CPU type.

Regardless of the chosen default, I'd keep the "'auto' is the fastest choice." text (unless we've proven that statement is no longer true).

Alternatively, leave the default alone, and we'll deal with it separately later.

Reply:
OK, changed.